express_relay/sdk/python/express_relay/svm/limo_client.py

```diff
@@ -44,14 +44,13 @@ class WSOLInstructions(TypedDict):
 
 
 class LimoClient:
-    def __init__(self, connection: AsyncClient, global_config: Pubkey):
+    def __init__(self, connection: AsyncClient):
         self._connection = connection
-        self._global_config = global_config
 
     async def get_all_orders_state_and_address_with_filters(
-        self, filters: List[MemcmpOpts]
+        self, filters: List[MemcmpOpts], global_config: Pubkey
```

Comment:
yeah i like the idea of specifying `global_config` as an argument to the filtering rather than defining it in the constructor. maybe could mention to adopt that convention for the typescript client also